packages/config/__snapshots__/index.spec.ts.js

```diff
@@ -7,7 +7,6 @@ exports['config/src/index .getBreakingKeys returns list of breaking config keys
   "experimentalRunEvents",
```

Comment:
**NOTE** above file is the store for all studio related things, straight copy paste from https://github.com/cypress-io/cypress/pull/23413/files#diff-c72280ee321ab528ae0e55a0f0d353cece65c1c115c60d89bf83e126b9fb3996

Feel free to take a look but I didn't implement this originally - happy to make improvements if you spot them, though. There's quite a bit going on, once this Studio feature is done we can spend some time on general "how it works", docs etc.